test/evaluation/submissionFrom.spec.ts

```diff
@@ -67,135 +66,132 @@ describe('Submission requirements tests', () => {
 
   it('Evaluate submission requirements max 2 from group B', () => {
     const pdSchema: PresentationDefinition = getFile('./test/resources/sr_rules.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp_general.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp_general.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.